### DISCUSSION TITLE
Tags showing "bad" quality in kepserver

### DISCUSSION THREAD

**Discussion by punarbhaglal** [eclipse-milo/milo#948] [General]
I am using a PLC simulator and trying to read tags of my program in TIA portal through Kepserver.
When I open quick client in kepserver to see values the quality is showing "**bad**", as I change the properties of Device, Data collection Enable and Simulated yes then the quality turns out to be good but the values are 0 instead of real values.

Please help me with a work around so I can read the values of my tags in kepserver. Below are the versions of software that I am using.
PLCSIM advanced 2.0 sp1 - plc that I choose is S7 1500
Kepserver Ex6
TIA portal v16

**Comment by kevinherron** [comment]
This question doesn’t seem to have anything to do with Milo, so it might be better suited for a generic PLC or industrial automation forum.

**Reply by punarbhaglal** [reply]
Okay thankyou so much for the reply
